### ISSUE TITLE
Publish deno-postgres on nest.land

### ISSUE THREAD

**Issue by martonlederer** [denodrivers/postgres#141]
[Nest.land](https://nest.land) is a growing ecosystem of Deno modules. It is a blockchain based package-registry, so if you publish a module, no one will be able to remove it ever again. This is especially useful and important regarding the url-based imports in Deno and most importantly, security. 
Even though Nest.land is in beta, it is really powerful and also faster than the github servers ([deno.land/x](https://deno.land/x)) just rewrites your url to github.

More information on https://nest.land/docs

**Comment by bombillazo** [comment]
Is nest.land still a thing?

**Comment by martonlederer** [comment]
Yup, nest.land still works, it is just not being actively worked on

**Comment by bombillazo** [comment]
We've added JSR support, you could contribute a PR to add support
